nix/pkgs/default.nix

```diff
@@ -99,7 +99,10 @@ let
 
   # not available in 20.03 and we depend on several recent changes
   # including stylish-haskell support
-  nix-pre-commit-hooks = import (sources."pre-commit-hooks.nix");
+  nix-pre-commit-hooks = (pkgs.callPackage ((sources."pre-commit-hooks.nix") + "/nix/default.nix") {
```

Comment:
Also this is worth a comment for posterity since it looks a bit weird.

Reply:
For sure. I added a comment explaining this weird stunt :+1: